weDevsOfficial/dokan
# enhance: update-admin-color-palette-settings-ui

### All Submissions:






### Changes proposed in this Pull Request:



Closes # .
Related with: https://github.com/weDevsOfficial/dokan-pro/pull/1830
Related with: https://github.com/weDevsOfficial/dokan/pull/1599

### How to test the changes in this Pull Request:

1. Enable "Color Scheme Customizer" module from admin.
2. Go to color tab from admin settings.
3. Now, here we see 9 types of wp (light, modern, ectoplasm, ocean etc.) template designs. Choose 1 of them.
4. Then go to the vendor dashboard and check the changes in the template design.



### Changelog entry

> Update admin color palette settings and design 9 types of vendor dashboard templates.


### PR Self Review Checklist:

* Code is not following code style guidelines
* Bad naming: make sure you would understand your code if you read it a few months from now.
* KISS: Keep it simple, Sweetie (not stupid!).
* DRY: Don't Repeat Yourself.
* Code that is not readable: too many nested 'if's are a bad sign.
* Performance issues
* Complicated constructions that need refactoring or comments: code should almost always be self-explanatory.
* Grammar errors.


### FOR PR REVIEWER ONLY:

> As a reviewer, your feedback should be focused on the idea, not the person. Seek to understand, be respectful, and focus on constructive dialog.

> As a contributor, your responsibility is to learn from suggestions and iterate your pull request should it be needed based on feedback. Seek to collaborate and produce the best possible contribution to the greater whole.
